internal/cloudapi/v2/v2_test.go

```diff
@@ -159,11 +159,11 @@ func TestCompose(t *testing.T) {
 		 }
 	}`, test_distro.TestArch3Name), http.StatusBadRequest, `
 	{
-		"href": "/api/image-builder-composer/v2/errors/4",
-		"id": "4",
+		"href": "/api/image-builder-composer/v2/errors/30",
+		"id": "30",
 		"kind": "Error",
-		"code": "IMAGE-BUILDER-COMPOSER-4",
-		"reason": "Unsupported distribution"
+		"code": "IMAGE-BUILDER-COMPOSER-30",
+		"reason": "Request could not be validated"
```

Comment:
This is suboptimal as it makes the error more generic. I would still prefer merging this and dealing with it in a follow-up (we should probably somehow propagate the whole validation error from kin-openapi to the response).

Reply:
Agreed, how the filter generates the error: https://github.com/getkin/kin-openapi/blob/master/openapi3filter/errors.go#L12-L39.